packages/react-core/src/components/Content/examples/ContentEditorial.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Content, ContentVariants } from '@patternfly/react-core';
+
+export const ContentEditorial: React.FunctionComponent = () => (
+  <>
+    <Content component={ContentVariants.h1} isEditorial>
+      Editorial content via components
```

Comment:
```suggestion
      Example of editorial content via components
```
Curious for thoughts on changing the title in the example. Because of how examples are styled on v6, I didn't realize I was reading the example at first, and was confused why there was a new heading